groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovyTextDocumentService.java

```diff
@@ -191,6 +205,15 @@ private void publishDiagnostics(Set<Diagnostic> diagnostics) {
         publishDiagnostics.accept(paramsBuilder.build());
     }
 
+    private Path getAbsolutePathRelativeToWorkspace(String uri) {
```

Comment:
the name of the method is a bit confusing since absolute path shouldn't be relative to anything... maybe resolveUriToWorkspaceRoot? in this case if uri is absolute the resolve will be just directly return it